src/librbd/internal.cc

```diff
@@ -689,15 +692,19 @@ int validate_pool(IoCtx &io_ctx, CephContext *cct) {
       api::Config<>::apply_pool_overrides(io_ctx, &config);
 
       uint32_t create_flags = 0U;
+      uint64_t mirror_image_mode = cls::rbd::MIRROR_IMAGE_MODE_JOURNAL;
```

Comment:
nit: it does not make difference, but I would use `RBD_MIRROR_IMAGE_MODE_JOURNAL` constant here. Because this is what users are going to use.